CMakeLists.txt

```diff
@@ -21,6 +21,14 @@ include_directories(include)
 
 ament_export_include_directories(include)
 
+add_library(${PROJECT_NAME} SHARED
```

Comment:
Instead of explicitly building a shared library this package should depend on [ament_cmake_ros](https://github.com/ros2/ament_cmake_ros/blob/05d9d3b9e8c0e51f55a25f54d7299013fc6e5b83/ament_cmake_ros/cmake/build_shared_libs.cmake) to use the CMake option choosing the library type.

Reply:
Done.